lib/tree-view.coffee

```diff
@@ -511,6 +505,20 @@ class TreeView extends View
     showProcess.onWillThrowError ({error, handle}) ->
       handle()
       handleError(error?.message)
+    showProcess
+
+  showSelectedEntryInFileManager: ->
+    return unless entry = @selectedEntry()
+
+    isFile = entry instanceof FileView
+    {command, args, label} = @fileManagerCommandForPath(entry.getPath(), isFile)
+    @openInFileManager(command, args, label, isFile)
+
+  showCurrentFileInFileManager: ->
+    return unless editor = atom.workspace.getActiveTextEditor()
+    return if editor.getPath() is undefined  # handle untitled tab
```

Comment:
will change to:

`return if not editor.getPath()`

to return early for both `null` and `undefined`